### PR TITLE
Add Taktikal Core connector

### PR DESCRIPTION
**Pull request by jobstaktikal**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by jobstaktikal** [comment]
@microsoft-github-policy-service agree [company="Taktikal"]

**Comment by jobstaktikal** [comment]
@microsoft-github-policy-service agree company="Taktikal"

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1818 @@
...
+          "$ref": "#/parameters/Accept"
+        }
+      ],
+      "get": {
+        "tags": [
+          "signing activity"
+        ],
+        "summary": "Gets activity log for Signing process",
+        "x-ms-summary": "Gets activity log for Signing process",
+        "description": "Get the the Activity Log for a Signing process. The data can be filtered filtered on number of parameters described for the function.",
```
Comment (vmanoharas) [inline certified-connectors/Taktikal Core/apiDefinition.swagger.json]: Hi @jobstaktikal  
Please remove repeated "filtered" word

Also, I would suggest you keep all parameters descriptions short to fit in the text box. please see [[here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters)]

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1818 @@
...
+          "$ref": "#/parameters/Accept"
+        }
+      ],
+      "post": {
+        "tags": [
+          "document sealing"
+        ],
+        "summary": "Seals a PDF document",
+        "x-ms-summary": "Seals a PDF document",
+        "description": "Seals a provided PDF document provided as a base64 string. \n\nA reason can be provided that will be displayed in the seal. \nAvalialbe languageType can be set to Is for Icelandic or En for English.",
```
Comment (vmanoharas) [inline certified-connectors/Taktikal Core/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1818 @@
...
+          "$ref": "#/parameters/Accept"
+        }
+      ],
+      "delete": {
+        "tags": [
+          "signing"
+        ],
+        "summary": "Cancel a Signing process",
+        "x-ms-summary": "Cancel a Signing process",
+        "description": "Cancel the Signing process based for a given Process Key. A user (email) can be provided that cancels the process.",
```
Comment (vmanoharas) [inline certified-connectors/Taktikal Core/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1818 @@
...
... After each signature the next signeee will get a message and so on until all have signed the document.\n* signatureLocation can be BottomLastPage or the default value TopFirstPage . Will control where to display the signature in the document.\n* meta is a key-value store that is used to store additional data for this particular signingProcess. The data here can be used to create the document if needed, and displayed in emails. All data in meta is also delivered in the webhook response.\n* flattenDocument Default value is false. If set to true the pdf document will be flatten.\n* reminderRule Sends reminders, leave empty or null for no reminders. The format should be \"{DAYS_AFTER_INITIAL_REMINDER}:{DAYS_BETWEEN_RECURRING_REMINDERS}\". e.g. \"1:3\" will send one reminder after 1 day and then every three days after that. Valid values will be for first reminder to be sent between 0-30 days (0 means no reminder) and then send every 0-15 days. 0, again, meaning no reminders. f.e 0:10 would only send reminders every 10 days and 5:0 would only send one reminder after 5 days\n\nDetailed documentation can be found on https://docs.taktikal.is/docs/api/using-the-signing-api/single-document",
```
Comment (vmanoharas) [inline certified-connectors/Taktikal Core/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/Taktikal Core/readme.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Taktikal Core
```
Comment (vmanoharas) [inline certified-connectors/Taktikal Core/readme.md:1]: please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

**Review by vmanoharas — COMMENTED** [review]
Hello @jobstaktikal,  

Could you please resolve review comments?

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json (hunk 1 above)
Comment (jobstaktikal) [reply certified-connectors/Taktikal Core/apiDefinition.swagger.json]: Updated

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json (hunk 2 above)
Comment (jobstaktikal) [reply certified-connectors/Taktikal Core/apiDefinition.swagger.json]: Updated

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json (hunk 3 above)
Comment (jobstaktikal) [reply certified-connectors/Taktikal Core/apiDefinition.swagger.json]: Updated

File: certified-connectors/Taktikal Core/apiDefinition.swagger.json (hunk 4 above)
Comment (jobstaktikal) [reply certified-connectors/Taktikal Core/apiDefinition.swagger.json]: Updated

File: certified-connectors/Taktikal Core/readme.md (hunk 5 above)
Comment (jobstaktikal) [reply certified-connectors/Taktikal Core/readme.md:1]: Updated

**Review by vmanoharas — APPROVED** [review]
Hello @jobstaktikal 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**